core/src/main/java/be/ida/medium/parser/MediumRssFeedParser.java

```diff
@@ -15,11 +15,13 @@
 
         for (SyndEntry syndEntry : syndFeed.getEntries()){
             MediumPost mediumPublication = new MediumPost();
+            int idStartPosition = syndEntry.getUri().lastIndexOf("rss----") + 1 ;
```

Comment:
Make use of StringUtils instead of filtering yourself.

For reference: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html